lab_6_pipeline/pipeline.py

```diff
@@ -44,37 +88,58 @@ def __init__(self, lemma: str = "", pos: str = "", tags: str = ""):
         """
         Initializes MorphologicalTokenDTO
         """
+        self.lemma = lemma
+        self.pos = pos
+        self.tags = tags
 
 
 class ConlluToken:
     """
     Representation of the CONLL-U Token
     """
 
-    def __init__(self, text: str):
+    def __init__(self, text: str, parameters: MorphologicalTokenDTO, position: int):
```

Comment:
you are not allowed to change API, discard

Reply:
done

Reply:
done

Reply:
do not see it in the code, have you committed and pushed?